File: ios/ChartViewManager.m
//
//  ChartViewManager.m
//  DietDoctorDemo
//
//  Created by Majid on 2/5/21.
//

#import <Foundation/Foundation.h>
#import "React/RCTViewManager.h"

@interface RCT_EXTERN_MODULE(ChartViewManager, RCTViewManager)
  RCT_EXPORT_VIEW_PROPERTY(data, NSArray)
  RCT_EXPORT_VIEW_PROPERTY(colors, NSArray)
  RCT_EXPORT_VIEW_PROPERTY(selectedIndex, NSNumber)
  RCT_EXPORT_VIEW_PROPERTY(onSelectedItem, RCTDirectEventBlock)
@end

File: ios/ChartViewManager.swift
//
//  ChartViewManager.swift
//  DietDoctorDemo
//
//  Created by Majid on 2/5/21.
//

import Foundation

@objc(ChartViewManager)
class ChartViewManager: RCTViewManager {
  override func view() -> UIView! {
    return ChartView()
  }
  
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
